packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -85,7 +130,11 @@ In the future when decorators become a native web platform feature, this may no
 
 ### Using decorators with TypeScript { #decorators-typescript }
```

Comment:
I think it's good we start off with our recommendation of using experimental decorator mode for TS, but would it be worthwhile to add a note, maybe at the bottom of this section, with regards to how to make alternative TS configs work?

Maybe inside a aside with warning, that to use standard decorator mode despite it's large emit, you must add `accessor` keyword on decorated class fields.

Also, if users for some reason _have_ to use `useDefineForClassFields: true`, they can do so either with or without `experimentalDecorators: true` by creating auto-accessors. Maybe this info belongs more in the "Avoiding issues with class fields" section.

Reply:
This is a great comment! Thank you. Regarding your comment about `useDefineForClassFields`, it prompted https://github.com/lit/lit.dev/issues/1243 which will be handled separately.

This did make me notice that no where on Lit.dev do we explicitly state our _hybrid decorator_ strategy. Not even in the upgrade guide. I've added a new incremental migration section to this PR that describes the hybrid decorator strategy.

I've added a section about